pom.xml

```diff
@@ -76,8 +76,8 @@
         <assertj.version>3.26.3</assertj.version><!-- @sync io.quarkus:quarkus-build-parent:${quarkus.version} prop:assertj.version -->
         <aws-java-sdk.version>1.11.714</aws-java-sdk.version>
         <azure-sdk-bom.version>${azure-sdk-bom-version}</azure-sdk-bom.version>
-        <azure-core.version>1.53.0</azure-core.version><!-- @sync com.azure:azure-sdk-bom:${azure-sdk-bom.version} dep:com.azure:azure-core -->
-        <azure-identity.version>1.14.0</azure-identity.version><!-- @sync com.azure:azure-sdk-bom:${azure-sdk-bom.version} dep:com.azure:azure-identity -->
+        <azure-core.version>1.54.1</azure-core.version><!-- @sync com.azure:azure-sdk-bom:${azure-sdk-bom.version} dep:com.azure:azure-core -->
```

Comment:
Also here, some autogen code. Please advise @jamesnetherton

Reply:
Nevermind, I've rebased and it fixed the code.

Reply:
Ah - I just realised you are targeting `camel-main`. Sorry, I had to force push that branch to get it ready for Camel 4.10.0. Hence the changes have leaked into this PR.

You should be able to target `main`, no? The changes you need should be in Camel 4.9.0?

Reply:
I used the `camel-main` as the component was not in the catalog for 4.9.0 (I did not add it by mistake), so, the extension plugin did not work correctly. However I understand the catalog was only needed to create the extension scaffold, is that correct? in such case I can try to do this PR against `main`.

Reply:
> I used the camel-main as the component was not in the catalog for 4.9.0 (I did not add it by mistake)

Ah, ok. If it's not in the catalog for 4.9.0, then I think some of our doc tooling would fail. So we have to keep targeting `camel-main`.